### PR TITLE
Harden the service by rejecting invalid messages from client

### PR DESCRIPTION
**Pull request by tanviraumi**
Tackles #5488. Will follow up with another PR with tests once service packages are published.

1. We are leveraging an invariant that any op sent by client should always have a valid client id. Currently, both r11s and push enforces this. Keeping that in mind, we introduced 'isServiceMessageType', which includes every op type that should only be sent by services. Since message sent by service does not have a clientId, we should be able to discard invalid ops quickly.

2. Removed code from deltaManager to promote certain op type data. That should not be necessary anymore.

3. Fixed 'sanitizeMessage' in Alfred. So even if client sends wrong fields, Alfred throws them away.

### CODE REVIEW

File: server/routerlicious/packages/lambdas/src/deli/lambda.ts [hunk 1]
```diff
@@ -484,6 +494,14 @@ export class DeliLambda implements IPartitionLambda {
         }
     }
 
+    private isInvalidMessage(message: IRawOperationMessage): boolean {
+        if (message.clientId) {
+            return isServiceMessageType(message.operation.type);
+        } else {
+            return true;
```
Comment (GaryWilber) [inline server/routerlicious/packages/lambdas/src/deli/lambda.ts:501]: Should this be `return false`?

Comment (tanviraumi) [reply server/routerlicious/packages/lambdas/src/deli/lambda.ts]: Correct!

**Review by curtisman — APPROVED** [review]
:shipit:

**Comment by curtisman** [comment]
I assume you will follow up with a PR adding tests once the server packages is published?

**Comment by tanviraumi** [comment]
> I assume you will follow up with a PR adding tests once the server packages is published?

Yes! I have a slightly modified version of your test passing. Need to add more test cases to it as well

**Review by GaryWilber — APPROVED** [review]

**Pull request merged**